tests/utils/validations.test.ts

```diff
@@ -72,4 +73,12 @@ describe('Test Address validations', () => {
     const result = validateSnAddress('')
     expect(result).toBe(false)
   })
+  it('return true if block number is a valid number', async () => {
+    const result = validateBlockNumber(123)
+    expect(result).toBe(true)
```

Comment:
Add tests for passing invalid (non-numeric) and null/undefined types, since your function handles that logic ;)